tests/utils.py

```diff
@@ -153,3 +156,14 @@ def get_noarch_python_meta(meta):
 
 def reset_config(search_path=None):
     reset_context(search_path)
+
+
+def raises_after(datetime_args, *args, **kwargs):
```

Comment:
Would this be better as a context manager that yields the values? That would make the intent of the utility more clear.

Reply:
This returns context managers, so it can be used as a context manager. See it in action at https://github.com/conda/conda-build/pull/5569/files#diff-ffb386b0dc58be25063999da06f74375b44a18685b726b2df7177f0d3bab8937R133

Reply:
Yep. I just mean making it one in the code would make people immediately understand that is how they should use it. Right now it looks like a function. Happy to pass though.

Reply:
I'll add a comment to make sure it works. I tried using `yield`s but then it becomes a double context manager.